src/generated/version.ts

```diff
@@ -0,0 +1 @@
+export const packageVersion = '13.7.0-post';
```

Comment:
that would be an invalid version. I think we should make sure that we only generate "release" versions